flood_forecast/train_model.py

```diff
@@ -1,8 +0,0 @@
-import argparse
```

Comment:
I am assuming you don't want a file that only does 1+1 so I deleted it...

Reply:
Yeah that can be removed.